### PR TITLE
Add slim-lint support

### LINKED ISSUE #923: Slim support
Would love [Slim](http://slim-lang.com/) support.

https://github.com/sds/slim-lint/blob/master/lib/slim_lint/linter/rubocop.rb

### PR DESCRIPTION
**Pull request by thorncp**
Also adds the `LintersJob` class and uses the `linters` queue.

https://github.com/houndci/linters/pull/105

Closes #923

### CODE REVIEW

**Comment by thorncp** [comment]
[Slim linter support](https://trello.com/c/SceZBl9f/805-slim-linter-support)

File: app/jobs/linters_job.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```
Comment (teoljungberg) [inline app/jobs/linters_job.rb]: Didn't we disable this rule?

File: spec/models/linter/slim_lint_spec.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+require "rails_helper"
+
+describe Linter::SlimLint do
+  it_behaves_like "a linter" do
+    let(:lintable_files) { %w(foo.slim) }
+    let(:not_lintable_files) { %w(foo.haml) }
+  end
+
+  describe "#file_review" do
```
Comment (teoljungberg) [inline spec/models/linter/slim_lint_spec.rb]: Could this be moved into the shared example group now?

File: app/jobs/linters_job.rb (hunk 1 above)
Comment (thorncp) [inline app/jobs/linters_job.rb]: ¯\\(°_o)/¯

File: spec/models/linter/slim_lint_spec.rb (hunk 2 above)
Comment (thorncp) [inline spec/models/linter/slim_lint_spec.rb]: Yes, but it'll take a bit of refactoring. I think that'll be better for another PR.

Comment (thorncp) [inline spec/models/linter/slim_lint_spec.rb]: Started in bd8da734278e1c5d7f64ccacc6ad78eea0fd3b9d

File: app/jobs/linters_job.rb (hunk 1 above)
Comment (gylaz) [inline app/jobs/linters_job.rb]: We disabled this in `thoughtbot/guides` and we are currently experiencing issues with org-level config.
